tests/e2e/config/bootstrap.js

```diff
@@ -172,6 +172,12 @@ const IGNORE_CONSOLE_MESSAGES = [
 		matcher: 'startsWith',
 		pattern: 'Warning: You are importing createRoot from',
 	},
+	// Gutenberg blockEditor.transformStyles warnings within the block editor.
```

Comment:
I created a follow up #11117 to investigate if this warning can be prevent by changing the way the E2E test image is setup rather than ignoring the warning.